File: apiEZPZ/src/main/java/com/example/apiezpz/checklist/dto/CategoryDTO.java
package com.example.apiezpz.checklist.dto;

import lombok.*;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CategoryDTO {
    private Long id;
    private String name;
    private Long checklistId; // 체크리스트 ID
    private List<ItemDTO> items;    //아이템 리스트 포함
}

File: apiEZPZ/src/main/kotlin/com/example/apiezpz/checklist/dto/CategoryDTO.kt
package com.example.apiezpz.checklist.dto

data class CategoryDTO(
    var id: Long? = null,
    var name: String = "",
    var checklistId: Long? = null, // 체크리스트 ID
    var items: List<ItemDTO> = emptyList()    // 아이템 리스트 포함
) 